gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/tasks/LaunchConfigTask.groovy

```diff
@@ -40,7 +40,8 @@ class LaunchConfigTask extends DefaultTask {
             '-XX:GCLogFileSize=10M',
             '-XX:NumberOfGCLogFiles=10',
             '-Xloggc:var/log/gc-%t-%p.log',
-            '-verbose:gc'
+            '-verbose:gc',
+            '-Dsun.net.inetaddr.ttl=20'
```

Comment:
Actually could we put this in a different place than `gcJvmOpts`?

Reply:
Yeah is there another list of options or just create a new variable?

Reply:
I think a new variable like `dnsJvmOpts` would be OK, they all get squashed together further down:

```
         writeConfig(createConfig(getArgs(), tmpdirJvmOpts + gcJvmOpts + defaultJvmOpts), getStaticLauncher())
```